docs/LayoutSpecs.json

```diff
@@ -49,10 +49,10 @@
                   "path": "DSCore.List.IsRectangular"
                 },
                 {
-                  "path": "TrueForAll"
+                  "path": "List.TrueForAll"
```

Comment:
@Benglin - Yes, these reflect the latest `LayoutSpecs.json` and `RawTypeData.json` in Dynamo.